### ISSUE TITLE
Wehre to kill all settings and courses?

### ISSUE THREAD

**Issue by MisterGeniusj007** [Stephan-S/FS19_AutoDrive#695]
Hallo,

i tried to kill al courses vie explorer/finder (Win- or Mac-System)!
So I found a xml in the savegame-Folder! I killed them!
But if I start the game there are the courses once again!
And, I can no more set direction-points!
If I'm doing it there is nothing to klick in the queue! No more names but spaces!
Can You explain where I will find all the settings for map and savegame?

Greetings an thanks!

**Comment by worker2419** [comment]
**close the game** and go to your Savegame folder, there is a file like AutoDrive_MapName_config.xml. Kill this file. Start your Game again. AD will create a new file for parameters without any Waypoint or Courses.

**Comment by Iwan1803** [comment]
And check the AD mod folder as there are some configs inside.

**Comment by MisterGeniusj007** [comment]
-> worker2419: I've done it! But if I started once again a got all the destination-points again in the menubar!
-> Iwan1803: is there anywhere such a folder? Never seen, only have the .zip-File!
So thank You! :-)

**Comment by Iwan1803** [comment]
Then take a look inside the zip file 👀

**Comment by worker2419** [comment]
look at screenshot, these Maps have init Config
![AD_01](https://user-images.githubusercontent.com/47278102/67631768-c711df00-f89a-11e9-812c-9ed00acc2680.png)

**Comment by MisterGeniusj007** [comment]
for both: Thanks a lot for Your answers!
I made a new version to save my problem:
I opened the AutoDrive_init_config.xml!
There I deleted the pathes and destianations!
And than . . tra tra .. I got a cleaned new AD-Version for the map!
The structure of the AD-config.xml looks the same as the config.xml at CS!
So I think there is no more problem to edit the AD-Savings!

But I'm extrem positiv about Your quick answers and helps,
that's why I want thank You very much again!

Greetings .... MG007
